plasma/plasma-mvp/morevp-alice-bob.md

```diff
@@ -0,0 +1,103 @@
+MoreVP Alice-Bob Scenarios
+---
+
+Note that these scenarios work with N participants, not just limited to 1 input 1 output. 
+
+Also, note that in all of these scenarios the exits are given the priority of the youngest input to the transaction.
+
+#### Alice & Bob are honest and cooperating:
+
+Happy path.
+
+1. Alice spends UTXO1 in TX1 to Bob.
+2. Operator withholds, neither Alice nor Bob know if the transaction has been included. 
+3. Someone with access to TX1 (Alice, Bob, or otherwise) starts an exit and places `exit bond`.
+4. Alice and Bob "piggyback" onto the exit and each put up `piggyback bond`.
+5. After 1 week, Bob receives the money as if the transaction were included. All bonds are refunded.
+
+
+#### Bob spends:
+
+Bob trying to cheat.
+
+1. Alice spends UTXO1 in TX1 to Bob, creating UTXO2.
+2. TX1 is included in block N.
+3. Bob spends UTXO2 in TX2.
+4. TX2 is included in block N+M.
+5. Bob starts an exit from TX1.
+6. Bob "piggybacks" onto the exit and places `piggyback bond`
```

Comment:
again - what if he doesn't piggyback - a no-op makes sense but just to be 100% clear. A no-op meaning Bob just made a useless transaction and game over?

What if Alice hasn't seen neither N nor N+M and piggybacks on that exit? Bob's piggyback will be slashed, but who get's the UTXOs in the end?

Reply:
If Bob doesn't piggyback, then Bob isn't eligible to get the money back (he won't anyway, because he cheated, but that's besides the point). 

If Alice piggybacks, then she'll be refunded at the end of the challenge period. No one gets to exit any UTXOs from the transaction because the transaction was valid, included, and spent later.